finder-app/Makefile

```diff
@@ -0,0 +1,30 @@
+INCLUDE_DIRS = 
+LIB_DIRS = 
+CFLAGS=-Wall -Werror
+#CC=aarch64-none-linux-gnu-gcc
+#CC=gcc
+
+# ifeq ($(TARGET),CROSS_COMPILE)
+#         CC = aarch64-none-linux-gnu-gcc
+# else
+#         CC = gcc
+# endif
+
+CC=$(CROSS_COMPILE)-gcc
```

Comment:
The cross compilation argument has a trailing "-". Refer to this instruction from Coursera.
Support for cross-compilation.  You should be able to generate an application for the native build platform when GNU make variable CROSS_COMPILE is not specified on the make command line.  When CROSS_COMPILE is specified with aarch64-none-linux-gnu- (note the trailing -)your makefile should compile successfully using the cross compiler installed in step 1.